packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeErrorRenderer.tsx

```diff
@@ -120,6 +130,7 @@ function ResultSetTooLarge({ errorNode, onFilterClick, limit, onOverrideLimit, s
   );
 }
 
+/** @alpha */
 interface LinkedNodeProps {
   errorNode: ErrorNode;
   scrollToElement: (node: ErrorNode) => void;
```

Comment:
- Why does this type has to take a node + a function that expects the same node as an argument? It seems either it shouldn't contain a node prop, or the function shouldn't take an argument.

- The type's name doesn't say anything about errors, but the attributes say it takes an "error node". Is this type intended only for errors, or can it be used for any kind of node? In case of the former, I think the type should be renamed. In case of the latter, I think the type should be changed to this:
   ```ts
   interface LinkedNodeProps<TNode> {
     node: TNode;
     scrollToNode: () => void;
   }
   ```

Reply:
Changed the structure of error renderer, this part was fixed.